### ISSUE TITLE
create OBO Library slice

### ISSUE THREAD

**Issue by graybeal** [ncbo/bioportal-project#284]
We have an OBO Foundry slice for the 9 or 10 ontologies that are approved in the Foundry. It could be nice to have an OBO Library slice that includes all the OBO Library ontologies.

**Comment by alexskr** [comment]
blocked by https://github.com/ncbo/ontologies_api/pull/87

**Comment by syphax-bouazzouni** [comment]
> blocked by [ncbo/ontologies_api#87](https://github.com/ncbo/ontologies_api/pull/87)

unblocked by https://github.com/ncbo/ontologies_api/pull/87#issuecomment-1783754744 (@alexskr)

**Comment by alexskr** [comment]
you are right; I mean to say that we are blocked until your PR gets merged.

**Comment by alexskr** [comment]
I discussed this with Chris and Naomi, and they shared the opinion that the distinction between OBO Foundry, which contains a set of well-established ontologies, and the OBO Library is no longer relevant. They suggest that instead of maintaining two separate slices, it would be more practical to include all the ontologies listed in OBO Foundry within the 'obo-foundry' slice.
